tests/db_engine_specs/mssql_tests.py

```diff
@@ -134,9 +140,9 @@ def test_column_datatype_to_string(self):
             (DATE(), "DATE"),
             (VARCHAR(length=255), "VARCHAR(255)"),
             (VARCHAR(length=255, collation="utf8_general_ci"), "VARCHAR(255)"),
-            (NVARCHAR(length=128), "NVARCHAR(128)"),
+            # (NVARCHAR(length=128), "NVARCHAR(128)"),
             (TEXT(), "TEXT"),
-            (NTEXT(collation="utf8_general_ci"), "NTEXT"),
+            # (NTEXT(collation="utf8_general_ci"), "NTEXT"),
```

Comment:
This functionality+test can be removed later, but I believe we need to make a db migration to resize the type column in the metadata (not sure if it was already).